.codecov.yml

```diff
@@ -0,0 +1,2 @@
+fixes:
+  - "ros_ws/src/sros2/::"
```

Comment:
This rewrites the path of the files in the report so that we can browse them directly on the codecov.io interface and see which lines/conditionals are covered and which are not